### ISSUE TITLE
The code needed to create a topography on the model 

### ISSUE THREAD

**Issue by hadismasihi** [cgre-aachen/gempy#655]
Hi
How can I create a topography similar to the actual topography of the area on the model?
Do I want it to have the same height, position and height as the opposite shape? And is this topography real or random?
![sssss](https://user-images.githubusercontent.com/63184005/143832430-09849bba-04a3-467e-8bd1-a0c976f52882.JPG)

**Comment by hadismasihi** [comment]
Why should the topography still be random, even though we are loading the elevation raster file in the code?
![ق](https://user-images.githubusercontent.com/63184005/143832515-27a8515a-e262-4e2a-802a-be857ee577b3.JPG)

**Comment by hadismasihi** [comment]
Hi, I need a quick answer to this question

**Comment by AlexanderJuestel** [comment]
Hello @hadismasihi sorry for not getting back to you earlier?

What are you modeling? Are you modelling a real world model or is it just an example model?

Cheers Alex

**Comment by hadismasihi** [comment]
Hello again
I modeled a real model.
![rmr2](https://user-images.githubusercontent.com/63184005/145680752-3dd214c0-ee2b-49a9-80db-5305deaeb49f.JPG)

**Comment by AlexanderJuestel** [comment]
Alright, do you have a topography for it? A digital elevation model? If so, you could load it as shown in the example notebooks or do you have any other issues?

**Comment by hadismasihi** [comment]
Yes, I have a digital elevation model and I influenced it in my model. But the topography is not right and a different topography is obtained with each run.
![NNNNNNNNNNNNNN](https://user-images.githubusercontent.com/63184005/145682924-a515ef82-bb46-4d4f-ac8f-c484414afde3.JPG)
![1گوگل](https://user-images.githubusercontent.com/63184005/145682949-e37631b1-7d09-48ba-b887-ad16ccf4ea4d.jpg)

**Comment by AlexanderJuestel** [comment]
How do you set the topography (code)?
Do the coordinates of your topography match the coordinates of your input points and orientations?

**Comment by hadismasihi** [comment]
Yes, the DEM is for the same area where the points were taken.
And I read on the site that these height models are random.
![یزیز](https://user-images.githubusercontent.com/63184005/145683782-4fb28480-cb95-442b-a9db-738cb64c0696.JPG)

**Comment by hadismasihi** [comment]
![333](https://user-images.githubusercontent.com/63184005/145684107-3d70ca75-ee2d-482c-81ad-0a070d8163e7.JPG)
![ننمئ](https://user-images.githubusercontent.com/63184005/145684149-effe4119-70ea-406d-8938-74d6e094cc8b.JPG)

**Comment by SimonVirgo** [comment]
Have you tried computeing the model after setting the topography and before plotting? Not sure if that is necessary...

**Comment by hadismasihi** [comment]
No, it is not necessary to add topography. But I just want to make sure that this software gives a random and unrealistic topography or that I am doing something wrong.

**Comment by AlexanderJuestel** [comment]
Hey again,

Do you want to have a random topography or the real topography in your model?

**Comment by hadismasihi** [comment]
Hello
Yes I want to have a real topography for my model.

**Comment by AlexanderJuestel** [comment]
Why do you add a random topography like here:

![image](https://user-images.githubusercontent.com/45469915/145718967-6fb90364-cb44-40c4-bf43-cae5916c3a77.png)

Just leave this out. I have the feeling that you overwrite the topography you have already set here: 

![image](https://user-images.githubusercontent.com/45469915/145718980-483bcc05-33f1-48c3-8d41-8555de828452.png)

**Comment by hadismasihi** [comment]
I'm sorry, I had a problem and I could not answer sooner.
How can I change this code and instead of "RANDOM" what should I write so that the model gives the topographic model based on my loaded altitude data?
![145718967-6fb90364-cb44-40c4-bf43-cae5916c3a77](https://user-images.githubusercontent.com/63184005/147402996-ab2aed1a-c896-447b-97d6-115b6aa5264a.png)

**Comment by AlexanderJuestel** [comment]
Hello, 

please see my example above how to set the topography based on a digital elevation model:

![image](https://user-images.githubusercontent.com/45469915/147405212-cf1ad6c1-2160-492d-9039-9f4d65a7463a.png)

**Comment by Japhiolite** [comment]
Any news here @hadismasihi ?
